test/integration/dynamic_settings.sh

```diff
@@ -130,12 +130,14 @@ cat <<EOT > /etc/clickhouse-server/config.d/storage_configuration_gcs.xml
     <disks>
       <disk_gcs>
         <type>s3</type>
-        <endpoint>https://storage.googleapis.com/${QA_GCS_OVER_S3_BUCKET}/clickhouse_backup_disk_gcs_over_s3/${HOSTNAME}/{cluster}/{shard}/</endpoint>
-        <access_key_id>${QA_GCS_OVER_S3_ACCESS_KEY}</access_key_id>
-        <secret_access_key>${QA_GCS_OVER_S3_SECRET_KEY}</secret_access_key>
+        <endpoint>http://gcs:8080/${QA_GCS_OVER_S3_BUCKET}/disk_gcs/{cluster}/{shard}/</endpoint>
```

Comment:
i think this is unnecessary, please rollback

Reply:
reverted, the only change in this file is it's now executable.